src/nvim/lib/kvec.h

```diff
@@ -77,10 +77,10 @@ int main() {
 		(v).items[(v).size++] = (x);										\
 	} while (0)
 
-#define kv_pushp(type, v) (((v).size == (v).capacity)?							\
+#define kv_pushp(type, v) ((((v).size == (v).capacity)?							\
 						   ((v).capacity = ((v).capacity? (v).capacity<<1 : 8),				\
 							(v).items = (type*)xrealloc((v).items, sizeof(type) * (v).capacity), 0)	\
-						   : 0), ((v).items + ((v).size++))
+						   : 0), ((v).items + ((v).size++)))
```

Comment:
if matchit.vim is with me today it is still needed. (without it, the entire macro won't be a single paren)